README.md

```diff
@@ -4,7 +4,7 @@ Alpha SOFIA
 ![Build Status](https://badge.buildkite.com/78b9e8a74bf2160a0d4cb7b72a17ee7d4a1590b3b3eaf847db.svg?branch=staging)
 [![Depfu](https://badges.depfu.com/badges/511b8df2e034f68fa0e7b3a4ac476094/overview.svg)](https://depfu.com/github/csvalpha/sofia?project_id=7740)
 
-The source code belonging to Alpha SOFIA. It is a system built with Ruby on Rails with Turbolinks and a little VueJS, used to manage orders in our own bar "Flux". Users authenticate via OAuth API (currently "Alpha Banana") to see how much credit they got left, or to be able to register new orders and/or payments.
+The source code belonging to Alpha SOFIA. It is a system built with Ruby on Rails with Turbolinks and a little VueJS, used to manage orders in our own bar "Flux". Users authenticate via OAuth API (currently "Alpha Amber") to see how much credit they got left, or to be able to register new orders and/or payments.
```

Comment:
Small thing but Amber is referred to in capital letters because it's an acronym (and Banana isn't), see for example amber-ui/amber-api repository titles and up here we also use SOFIA instead of Sofia